internal/addon/var.go

```diff
@@ -31,9 +31,14 @@ const (
 	InstrumentationResource         = "instrumentations"
 	SpokeOTELColName                = "mcoa-instance"
 	SpokeInstrumentationName        = "mcoa-instance"
+	SpokeUIPluginName               = "monitoring"
```

Comment:
I'm also fine with `MonitoringUIPluginName` removed the spoke part since now this will also install on the hub
```suggestion
	IDetectionUIPluginName               = "monitoring"
```